src/pages/media.tsx

```diff
@@ -0,0 +1,160 @@
+    eventLink:
+      "https://www.mpa.gov.sg/web/portal/home/maritime-singapore/what-maritime-singapore-offers/maritime-perspectives-webinar-series/digital-connectivity-%26-data-standards",
+  },
+  {
+    title: "Joining forces for trade digitisation: SWIFT, the United Nations, and the Singapore Government",
+    dateTime: "5 Oct | 4pm - 4.30pm SGT",
+    description:
+      "The Covid-19 pandemic has magnified the need for trade digitisation. With widespread working from home; supply chain, travel, and delivery disruptions; and couriers struggling to get through to lockdown locations in a timely way, there’s never been a greater catalyst to go digital. Watch this session to find out how SWIFT is taking the global trade digitisation effort one step further, in collaborating across the ecosystem.",
+    watchLink: "https://www.swift.com/swift-at-sibos/joining-forces-trade-digitisation",
+    eventLink: "https://www.swift.com/swift-at-sibos/joining-forces-trade-digitisation",
+  },
+];
+
+export const MediaPage = () => (
+  <>
+    <Helmet>
+      <meta property="description" content="These are media events which TradeTrust has been involved in." />
+      <meta property="og:description" content="These are media events which TradeTrust has been involved in." />
+      <meta property="og:title" content="TradeTrust - An easy way to check and verify your documents" />
+      <meta property="og:url" content={`${window.location.origin}/media`} />
+      <title>TradeTrust - Events and Media Centre</title>
+    </Helmet>
+
+    <ResourcePageStyle>
+      <div className="bg-blue-lighter">
+        <div className="container-custom py-5">
+          <div className="row">
+            <div className="col-12">
+              <h1>Events and Media Centre</h1>
+            </div>
+          </div>
+          <div className="row py-3">
+            <div className="col-12 col-md-12 col-lg-4 order-lg-2">
+              <div className="side-links">
+                {sideLinks.map((resourceLink, index) => (
+                  <ResourcesLink
+                    key={index}
+                    title={resourceLink.title}
+                    type={resourceLink.type}
+                    details={resourceLink.details}
+                    icon={resourceLink.icon}
+                  />
+                ))}
+              </div>
+            </div>
+            <div className="col-12 col-md-12 col-lg-8 order-lg-1">
+              {resources.map((resource, index) => (
+                <ResourcesCard details={resource} key={index} />
+              ))}
+            </div>
+          </div>
+        </div>
+      </div>
+    </ResourcePageStyle>
+  </>
+);
+
+const ResourcePageStyle = styled.div`
+  .side-links {
+    @media (min-width: ${vars.lg}) {
+      position: sticky;
+      top: 10px;
+    }
+  }
```

Comment:
for this page i think no need (content too tall already), alternatively u can make the sidebar scrollable but i'll prefer to drop sticky altogether can already

Reply:
Ahh okie!